### DISCUSSION TITLE
Keyboard shortcuts for outfits not working as intended?

### DISCUSSION THREAD

**Discussion by brendanjones** [endless-sky/endless-sky#7184] [Balance, Mechanics, and Interactions]
This is as much a question as a suggestion as maybe I'm just getting keyboard shortcuts wrong:

When in outfits, and you have an item both in shop stock and in either cargo or storage, shouldn't
1. I (Install) source from your cargo or storage, and
2. B (Buy) source from your shop stock?

Currently if I have stock in both, and I press B, the outfit is taken from my cargo or storage first instead of the shop stock first.

**Comment by TomGoodIdea** [comment]
With #10743 merged, the UI should be more clear now.
